package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "esnextbin",
-  "version": "0.1.0",
+  "version": "1.0.0-alpha.0",
```

Comment:
@jameswomack what's the point of changing this? I feel completely fine with `0` version for now.